packages/consoledot-api/src/fetchers/fetchKafkaMetrics.ts

```diff
@@ -54,9 +73,33 @@ export async function fetchKafkaMetrics({
       );
     }
 
+    function addAggregateBrokerBytes() {
+      console.log(broker_id);
+      const broker =
+        usedDiskSpaceMetrics[broker_id] || usedDiskSpaceMetrics["2"];
```

Comment:
Can we have a comment to describe this magic "2" value? It's not clear to me why we have it by looking at this.